earthpy/tests/test_plot.py

```diff
@@ -16,7 +15,7 @@
 
 # Create tuple
 tuups = (1, 2)
-im = np.random.randint(10, size=(2, 4, 5))
+im_arr = np.random.randint(10, size=(2, 4, 5))
```

Comment:
Instead of defining this as a module-level variable, it would be better to use a pytest fixture, which ensures that each test function gets it's own version of the object (and in this case, different randomly generated numbers). You could do something like: 

```python
@pytest.fixture
def image_array_3d():
    return np.random.randint(10, size=(2, 4, 5))
```

Reply:
ok i think i've fixed this throughout this test module.